### ISSUE TITLE
What Kids Need to Learn to Succeed in 2050  Youth Now  Medium

### ISSUE THREAD

**Issue by guevara** [guevara/read-it-later#316]
What Kids Need to Learn to Succeed in 2050 – Youth, Now – Medium<br>
<br>
https://ift.tt/2xfyk0z<br>
<br>
Yuval Noah Harari<br>
<br>

<div><div>
<section name="4c16"><div><hr></div>
<div><div>
<p name="bc79" id="bc79"><span>H</span>umankind is facing unprecedented revolutions, all our old stories are crumbling, and no new story has so far emerged to replace them. How can we prepare ourselves and our children for a world of such unprecedented transformations and radical uncertainties? A baby born today will be thirtysomething in 2050. If all goes well, that baby will still be around in 2100 and might even be an active citizen of the 22nd century. What should we teach that baby that will help them survive and flourish in the world of 2050 or the 22nd century? What kind of skills will they need in order to get a job, understand what is happening around them, and navigate the maze of life?</p>
<p name="e987" id="e987">Unfortunately, since nobody knows what the world will look like in 2050 — not to mention 2100 — we don’t know the answer to these questions. Of course, humans have never been able to predict the future with accuracy. But today it is more difficult than ever before because once technology enables us to engineer bodies, brains, and minds, we will no longer be able to be certain about anything — including things that previously seemed fixed and eternal.</p>
<p name="bcdd" id="bcdd">A thousand years ago, in 1018, there were many things people didn’t know about the future, but they were nevertheless convinced that the basic features of human society were not going to change. If you lived in China in 1018, you knew that by 1050 the Song Empire might collapse, the Khitans might invade from the north, and plagues might kill millions. However, it was clear to you that even in 1050 most people would still work as farmers and weavers, rulers would still rely on humans to staff their armies and bureaucracies, men would still dominate women, life expectancy would still be about 40, and the human body would remain exactly the same. For that reason, in 1018 poor Chinese parents taught their children how to plant rice or weave silk; wealthier parents taught their boys how to read the Confucian classics, write calligraphy, or fight on horseback, and they taught their girls to be modest and obedient housewives. It was obvious that these skills would still be needed in 1050.</p>
<blockquote name="cc04" id="cc04">To keep up with the world of 2050, you will need to do more than merely invent new ideas and products, but above all, reinvent yourself again and again.</blockquote>
<p name="b1bc" id="b1bc">In contrast, today we have no idea how China or the rest of the world will look in 2050. We don’t know what people will do for a living, we don’t know how armies or bureaucracies will function, and we don’t know what gender relations will be like. Some people will probably live much longer than today, and the human body itself might undergo an unprecedented revolution, thanks to bioengineering and direct brain-to-computer interfaces. Much of what kids learn today will likely be irrelevant by 2050.</p>
<p name="fb83" id="fb83">At present, too many schools focus on cramming information into kids’ brains. In the past, this made sense, because information was scarce and even the slow trickle of existing information was repeatedly blocked by censorship. If you lived, say, in a small provincial town in Mexico in 1800, it was difficult for you to know much about the wider world. There was no radio, television, daily newspaper, or public library. Even if you were literate and had access to a private library, there was not much to read other than novels and religious tracts. The Spanish empire heavily censored all texts printed locally and allowed only a dribble of vetted publications to be imported from the outside. Much the same was true if you lived in some provincial town in Russia, India, Turkey, or China. When modern schools came along, teaching every child to read and write and imparting the basic facts of geography, history, and biology, they represented an immense improvement.</p>
<p name="0872" id="0872">In contrast, in the 21st century, we are flooded with enormous amounts of information, and the censors don’t even try to block it. Instead, they are busy spreading misinformation or distracting us with irrelevancies. If you live in some provincial Mexican town and have a smartphone, you can spend many lifetimes just reading Wikipedia, watching TED Talks, and taking free online courses. No government can hope to conceal all the information it doesn’t like. On the other hand, it is alarmingly easy to inundate the public with conflicting reports and red herrings. People all over the world are but a click away from the latest accounts of the bombardment of Aleppo or melting ice caps in the Arctic, but there are so many contradictory accounts that it is hard to know what to believe. Besides, countless other things are just a click away as well, making it difficult to focus, and when politics or science look too complicated, it is tempting to switch to some funny cat videos, celebrity gossip, or porn.</p>
<p name="bfd2" id="bfd2">In such a world, the last thing a teacher needs to give her pupils is more information. They already have far too much of it. Instead, people need the ability to make sense of information, to tell the difference between what is important and what is unimportant, and, above all, to combine many bits of information into a broad picture of the world.</p>
<p name="b8a2" id="b8a2">In truth, this has been the ideal of Western liberal education for centuries, but up until, now even many Western schools have been rather slack in fulfilling it. Teachers allowed themselves to focus on imparting data while encouraging students “to think for themselves.” Due to their fear of authoritarianism, liberal schools have had a particular horror of grand narratives. They’ve assumed that as long as we give students lots of data and a modicum of freedom, the students will create their own picture of the world, and even if this generation fails to synthesize all the data into a coherent and meaningful story about the world, there will be plenty of time to construct a better synthesis in the future.</p>
<p name="b6ca" id="b6ca">We have now run out of time. The decisions we will make in the next few decades will shape the future of life itself, and we can make these decisions based only on our present worldview. If this generation lacks a comprehensive view of the cosmos, the future of life will be decided at random.</p>
<h3 name="de95" id="de95">The Heat Is On</h3>
<p name="90bd" id="90bd">Besides information, most schools also focus too much on providing students with a set of predetermined skills, such as solving differential equations, writing computer code in C++, identifying chemicals in a test tube, or conversing in Chinese. Yet since we have no idea what the world and the job market will look like in 2050, we don’t really know what particular skills people will need. We might invest a lot of effort teaching kids how to write in C++ or speak Chinese, only to discover that by 2050, artificial intelligence can code software far better than humans and a new Google Translate app will enable you to conduct a conversation in almost flawless Mandarin, Cantonese, or Hakka, even though you only know how to say “<em>Ni hao</em>.”</p>
<p name="bae0" id="bae0">So, what should we be teaching? Many pedagogical experts argue that schools should switch to teaching “the four Cs” — critical thinking, communication, collaboration, and creativity. More broadly, they believe, schools should downplay technical skills and emphasize general-purpose life skills. Most important of all will be the ability to deal with change, learn new things, and preserve your mental balance in unfamiliar situations. To keep up with the world of 2050, you will need to do more than merely invent new ideas and products, but above all, reinvent yourself again and again.</p>
<blockquote name="d18c" id="d18c">If somebody describes the world of the mid-21st century to you and it <em>doesn’t </em>sound like science fiction, it is certainly false.</blockquote>
<p name="413b" id="413b">For as the pace of change increases, not just the economy, but the very meaning of “being human” is likely to mutate. Already in 1848, the <em>Communist Manifesto </em>declared that “all that is solid melts into air.” Marx and Engels, however, were thinking mainly about social and economic structures. By 2048, physical and cognitive structures will also melt into air, or into a cloud of data bits.</p>
<p name="ef00" id="ef00">In 1848, millions of people were losing their jobs on village farms and going to the big cities to work in factories. But upon reaching the big city, they were unlikely to change their gender or add a sixth sense. And if they found a job in some textile factory, they could expect to remain in that profession for the rest of their working lives.</p>
<p name="fdef" id="fdef">By 2048, people might have to cope with migrations to cyberspace, fluid gender identities, and new sensory experiences generated by computer implants. If they find both work and meaning in designing up-to-the-minute fashions for a 3D virtual reality game, within a decade, not just this particular profession, but all jobs demanding this level of artistic creation might be taken over by A.I. So, at age 25, you might introduce yourself on a dating site as “a 25-year-old heterosexual woman who lives in London and works in a fashion shop.” At 35, you might say you are “a gender-nonspecific person undergoing age adjustment, whose neocortical activity takes place mainly in the NewCosmos virtual world, and whose life mission is to go where no fashion designer has gone before.” At 45, both dating and self-definitions are passé. You just wait for an algorithm to find (or create) the perfect match for you. As for drawing meaning from the art of fashion design, you are so irrevocably outclassed by the algorithms that looking at your crowning achievements from the previous decade fills you with embarrassment rather than pride. And you still have many decades of radical change ahead of you.</p>
<p name="af6a" id="af6a">Please don’t take this scenario literally. Nobody can predict the specific changes we will witness in the future. Any particular scenario is likely to be far from the truth. If somebody describes the world of the mid-21st century to you and it sounds like science fiction, it is probably false. But then again, if somebody describes the world of the mid-21st century to you and it <em>doesn’t </em>sound like science fiction, it is certainly false. We cannot be sure of the specifics; change itself is the only certainty.</p>
<p name="39b4" id="39b4">Such profound change may well transform the basic structure of life, making discontinuity its most salient feature. From time immemorial, life was divided into two complementary parts: a period of learning followed by a period of working. In the first part of life, you accumulated information, developed skills, constructed a worldview, and built a stable identity. Even if at 15 you spent most of your day working in your family’s rice field (rather than in a formal school), the most important thing you were doing was learning: how to cultivate rice, how to conduct negotiations with the greedy rice merchants from the big city, and how to resolve conflicts over land and water with the other villagers. In the second part of life, you relied on your accumulated skills to navigate the world, earn a living, and contribute to society. Of course, even at 50, you continued to learn new things about rice, merchants, and conflicts, but these were just small tweaks to your well-honed abilities.</p>
<p name="ddb3" id="ddb3">By the middle of the 21st century, accelerating change plus longer lifespans will make this traditional model obsolete. Life will come apart at the seams, and there will be less and less continuity between different periods of life. “Who am I?” will be a more urgent and complicated question than ever before.</p>
<p name="afc6" id="afc6">This is likely to involve immense levels of stress. Change is almost always stressful, and after a certain age most people just don’t like to do it. When you are 15, your entire life is change. Your body is growing, your mind is developing, your relationships are deepening. Everything is in flux, and everything is new. You are busy inventing yourself. Most teenagers find it frightening, but at the same time, it is also exciting. New vistas are opening before you, and you have an entire world to conquer.</p>
<p name="0b7f" id="0b7f">By the time you are 50, you don’t want change, and most people have given up on conquering the world. Been there, done that, got the T-shirt. You prefer stability. You have invested so much in your skills, your career, your identity, and your worldview that you don’t want to start all over again. The harder you’ve worked on building something, the more difficult it is to let go of it and make room for something new. You might still cherish new experiences and minor adjustments, but most people in their 50s aren’t ready to overhaul the deep structures of their identity and personality.</p>
<p name="4cbf" id="4cbf">There are neurological reasons for this. Though the adult brain is more flexible and volatile than was once thought, it is still less malleable than the teenage brain. Reconnecting neurons and rewiring synapses is hard work. But in the 21st century, you can’t afford stability. If you try to hold on to some stable identity, job, or worldview, you risk being left behind as the world flies by you with a whoosh. Given that life expectancy is likely to increase, you might subsequently have to spend many decades as a clueless fossil. To stay relevant — not just economically but above all socially — you will need the ability to constantly learn and to reinvent yourself, certainly at a young age like 50.</p>
<blockquote name="12de" id="12de">The best advice I can give a 15-year-old is: don’t rely on the adults too much. Most of them mean well, but they just don’t understand the world.</blockquote>
<p name="cf20" id="cf20">As strangeness becomes the new normal, your past experiences, as well as the past experiences of the whole of humanity, will become less reliable guides. Humans as individuals and humankind as a whole will increasingly have to deal with things nobody ever encountered before, such as super-intelligent machines, engineered bodies, algorithms that can manipulate emotions with uncanny precision, rapid man-made climate cataclysms, and the need to change your profession every decade. What is the right thing to do when confronting a completely unprecedented situation? How should you act when you are flooded by enormous amounts of information and there is absolutely no way you can absorb and analyze it all? How do you live in a world where profound uncertainty is not a bug but a feature?</p>
<p name="84e5" id="84e5">To survive and flourish in such a world, you will need a lot of mental flexibility and great reserves of emotional balance. You will have to repeatedly let go of some of what you know best, and learn to feel at home with the unknown. Unfortunately, teaching kids to embrace the unknown while maintaining their mental balance is far more difficult than teaching them an equation in physics or the causes of the First World War. You cannot learn resilience by reading a book or listening to a lecture. Teachers themselves usually lack the mental flexibility that the 21st century demands since they themselves are the product of the old educational system.</p>
<p name="94fa" id="94fa">The Industrial Revolution has bequeathed us the production-line theory of education. In the middle of town, there is a large concrete building divided into many identical rooms, each room equipped with rows of desks and chairs. At the sound of a bell, you go to one of these rooms together with 30 other kids who were all born the same year as you. Every hour a different grown-up walks in and starts talking. The grown-ups are all paid to do so by the government. One of them tells you about the shape of the earth, another tells you about the human past, and a third tells you about the human body. It is easy to laugh at this model, and almost everybody agrees that no matter its past achievements, it is now bankrupt. But so far we haven’t created a viable alternative. Certainly not a scalable alternative that can be implemented in rural Mexico rather than just in wealthy California suburbs.</p>
<h3 name="0d7f" id="0d7f">Hacking Humans</h3>
<p name="429d" id="429d">So the best advice I can give a 15-year-old stuck in an outdated school somewhere in Mexico, India, or Alabama is: don’t rely on the adults too much. Most of them mean well, but they just don’t understand the world. In the past, it was a relatively safe bet to follow the adults, because they knew the world quite well, and the world changed slowly. But the 21st century is going to be different. Because of the increasing pace of change, you can never be certain whether what the adults are telling you is timeless wisdom or outdated bias.</p>
<p name="b779" id="b779">So on what can you rely instead? Perhaps on technology? That’s an even riskier gamble. Technology can help you a lot, but if technology gains too much power over your life, you might become a hostage to its agenda. Thousands of years ago humans invented agriculture, but this technology enriched just a tiny elite while enslaving the majority of humans. Most people found themselves working from sunrise till sunset plucking weeds, carrying water buckets, and harvesting corn under a blazing sun. It could happen to you too.</p>
<p name="9ab4" id="9ab4">Technology isn’t bad. If you know what you want in life, technology can help you get it. But if you don’t know what you want in life, it will be all too easy for technology to shape your aims for you and take control of your life. Especially as technology gets better at understanding humans, you might increasingly find yourself serving it, instead of it serving you. Have you seen those zombies who roam the streets with their faces glued to their smartphones? Do you think they control the technology, or does the technology control them?</p>
<p name="696e" id="696e">Should you rely on yourself, then? That sounds great on <em>Sesame Street </em>or in an old-fashioned Disney film, but in real life, it doesn’t work so well. Even Disney is coming to realize it. Just like Riley Andersen, most people barely know themselves, and when they try to “listen to themselves” they easily become prey to external manipulations. The voice we hear inside our heads is never trustworthy because it always reflects state propaganda, ideological brainwashing, and commercial advertisements, not to mention biochemical bugs.</p>
<p name="2731" id="2731">As biotechnology and machine learning improve, it will become easier to manipulate people’s deepest emotions and desires, and it will become more dangerous than ever to just follow your heart. When Coca-Cola, Amazon, Baidu, or the government knows how to pull the strings of your heart and press the buttons of your brain, will you still be able to tell the difference between your self and their marketing experts?</p>
<blockquote name="dc1c" id="dc1c"><strong>If you don’t know what you want in life, it will be all too easy for technology to shape your aims for you and take control of your life.</strong></blockquote>
<p name="ffe6" id="ffe6">To succeed at such a daunting task, you will need to work very hard at getting to know your operating system better — to know what you are and what you want from life. This is, of course, the oldest advice in the book: know thyself. For thousands of years, philosophers and prophets have urged people to know themselves. But this advice was never more urgent than in the 21st century, because unlike in the days of Laozi or Socrates, now you have serious competition. Coca-Cola, Amazon, Baidu, and the government are all racing to hack you. Not your smartphone, not your computer, and not your bank account; they are in a race to hack <em>you </em>and your organic operating system. You might have heard that we are living in the era of hacking computers, but that’s not even half the truth. In fact, we are living in the era of hacking humans.</p>
<p name="ce93" id="ce93">The algorithms are watching you right now. They are watching where you go, what you buy, who you meet. Soon they will monitor all your steps, all your breaths, all your heartbeats. They are relying on Big Data and machine learning to get to know you better and better. And once these algorithms know you better than you know yourself, they can control and manipulate you, and you won’t be able to do much about it. You will live in the matrix, or in <em>The Truman Show</em>. In the end, it’s a simple empirical matter: if the algorithms indeed understand what’s happening within you better than you understand it yourself, authority will shift to them.</p>
<p name="b058" id="b058">Of course, you might be perfectly happy ceding all authority to the algorithms and trusting them to decide things for you and for the rest of the world. If so, just relax and enjoy the ride. You don’t need to do anything about it. The algorithms will take care of everything. If, however, you want to retain some control over your personal existence and the future of life, you have to run faster than the algorithms, faster than Amazon and the government, and get to know yourself before they do. To run fast, don’t take much baggage with you. Leave all your illusions behind. They are very heavy.</p>
</div></div></section><section name="3299"><div><hr></div>
<div><div><figure name="2920" id="2920"><div>
<div></div>
<a href="https://www.penguinrandomhouse.com/books/579741/21-lessons-for-the-21st-century-by-yuval-noah-harari/" rel="nofollow"><div>
<img src="https://cdn-images-1.medium.com/freeze/max/1000/1*SsxmnxtjYN-fuM_h8Z7m1Q@2x.png?q=20"><canvas></canvas><img><img src="https://cdn-images-1.medium.com/max/1000/1*SsxmnxtjYN-fuM_h8Z7m1Q@2x.png">
</div></a>
</div>
<figcaption>From the book <a href="https://www.penguinrandomhouse.com/books/579741/21-lessons-for-the-21st-century-by-yuval-noah-harari/" rel="noopener nofollow" target="_blank">21 LESSONS FOR THE 21ST CENTURY</a> by Yuval Noah Harari. Copyright © 2018 by Yuval Noah Harari. Published by Spiegel &amp; Grau, an imprint of Random House, a division of Penguin Random House LLC. All rights reserved.</figcaption></figure></div></div></section>
</div></div><br>
<br>
<br>
<br>
<br>
<br>
via Medium https://medium.com<br>
<br>
September 28, 2018 at 03:01PM
